content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -8,14 +8,24 @@ test: yes
 ---
 
 In addition to capturing application traffic, Istio can also capture DNS requests to improve the performance and usability of your mesh.
-When proxying DNS, all DNS requests from an application will be redirected to the sidecar, which stores a local mapping of domain names to IP addresses. If the request can be handled by the sidecar, it will directly return a response to the application, avoiding a roundtrip to the upstream DNS server. Otherwise, the request is forwarded upstream following the standard `/etc/resolv.conf` DNS configuration.
+When proxying DNS, all DNS requests from an application will be redirected to the sidecar or ztunnel proxy, which stores a local mapping of domain names to IP addresses. If the request can be handled by the proxy, it will directly return a response to the application, avoiding a roundtrip to the upstream DNS server. Otherwise, the request is forwarded upstream following the standard `/etc/resolv.conf` DNS configuration.
 
 While Kubernetes provides DNS resolution for Kubernetes `Service`s out of the box, any custom `ServiceEntry`s will not be recognized. With this feature, `ServiceEntry` addresses can be resolved without requiring custom configuration of a DNS server. For Kubernetes `Service`s, the DNS response will be the same, but with reduced load on `kube-dns` and increased performance.
 
 This functionality is also available for services running outside of Kubernetes. This means that all internal services can be resolved without clunky workarounds to expose Kubernetes DNS entries outside of the cluster.
 
 ## Getting started
 
+In sidecar mode, all traffic passes through Layer 7 processing, and the `hosts` of a `ServiceEntry` can be matched to a HTTP header without requiring the use of DNS. In ambient mode, the ztunnel only sees traffic at Layer 4, and does not have access to HTTP headers. Therefore, DNS proxying is required to enable resolution of `ServiceEntry` addresses, especially in the case of [sending egress traffic to waypoints](https://github.com/istio/istio/wiki/Troubleshooting-Istio-Ambient#scenario-ztunnel-is-not-sending-egress-traffic-to-waypoints).
```

Comment:
I think this is fine but to nit pick a little:
Technically sidecars have DNS proxy capability because not all traffic is suitable for this form of routing and that is perhaps unclear in this paragraph. We could consider adding "If routing based on HTTP header is not possible (TCP traffic for instance), DNS proxy can be enabled." or something to this effect but written more clearly.